packages/db/postgres/tables.sql

```diff
@@ -241,3 +241,12 @@ CREATE TABLE IF NOT EXISTS name
     inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
     updated_at  TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS pinning_authorisation
```

Comment:
For NFT.Storage the plan is to have user "tags" and have the `getUser` call join on this table so a separate request does not have to be made. It also allows multiple features/permissions for a given user.

For example:

| User ID | Tags            |
|---------|-----------------|
| 138     | psa             |
| 567     | psa,storage:2TB |
| 875     | storage:10TB    |

This is fine for now though and can be easily refactored and migrated.

Reply:
@jsdevel can you verify what I've said above is accurate?